### DISCUSSION TITLE
Any update on server api/routes?

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [nuxt/devtools#421] [Q&A] [answered]
Any update on server api/routes?
Like being able to see the server api/routes and add new ones, and make requests like postman?

I read once that was a feature being worked on.

Everyday more hyped for Nuxt! Great job to the team. 🔥🤯🤝🎉

**Comment by arashsheyda** [comment, marked as answer]
You can already see the routes from server `api` & `routes` and send requests like postman:
![image](https://github.com/nuxt/devtools/assets/38922203/cffa6fe9-214c-44de-8285-b579a352c1d4)

and for adding new routes do you mean like you want to be able to create route from DevTools Server Rotues tab directly?

**Reply by angelhdzmultimedia** [reply]
Thank you so much for that quick and kind response!

What I meant about being able to create routes/apis from the Devtools, I remembered the Nuxtr VSCode extension already allows us to create them, and create pages, middleware, components, etc.

I'm so going to check the Postman-like feature right now! 

Wishing you health, peace, and success.

**Reply by arashsheyda** [reply]
thank you! Yes that's a good idea. I whish you the same <3

**Comment by angelhdzmultimedia** [comment]
> You can already see the routes from server `api` & `routes` and send requests like postman: ![image](https://user-images.githubusercontent.com/38922203/268739873-cffa6fe9-214c-44de-8285-b579a352c1d4.png)
> 
> and for adding new routes do you mean like you want to be able to create route from DevTools Server Rotues tab directly?
